File: yunj/level1/21_String_Handling_Basic.java
import java.util.regex.Pattern;

class Solution {
    public boolean solution(String s) {
        if (s.length() != 4 && s.length() != 6) {
            return false;
        }
        String pattern = "^[0-9]*$";
        return Pattern.matches(pattern, s);
    }
}

File: sangjin/level1/21_string_handling_basic.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    
    if(s.size() == 4 || s.size() == 6){
        for(int i = 0; i < s.size(); i++){
            if(!(s[i] >= '0' && s[i] <= '9')){
                return false;
            }
        }
        return true;
    }
    
    return false; 
}